### PR TITLE
dash/datagrid2-tests

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed and improved DataGrid tests.

### CODE REVIEW

**Review by ddragula — APPROVED** [review]

**Comment by sebastianbochan** [comment]
It will not be valid with the newest Datagrid. The question is: should we merge it and replace all references afterward or do it here (after the merge with the current `datagrid component` branch).

I'm more convinced to the second option.

**Review by ddragula — COMMENTED** [review]
It's OK, but I'll hold off on approving it for now completely. It's best to link `dash/datagrid2-component` here, resolve any conflicts and add tests for the component here as well.

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**
